### ISSUE TITLE
Simple Moving Average SMA44 rising and falling 

### ISSUE THREAD

**Issue by rahulmr** [kieran-mackle/autotrader-demo#3]
**Describe the strategy you'd like to see added**
While SMA 44 is rising (last 3-5 candles) on 5 min timeframe if green candle crosses and closes above the 44SMA, then buy order must be placed above high (with buffer like 0.2) of that (crossing) candle and stop loss is loss(with some buffer) of same candle.  
Target must be 1:4 and Quantity must be calculated based on risk per trade.

Similar steps to be done on sell side.

**Is your strategy request related to one posted online somewhere? Please provide any links.**
Will add more details later

Basically, candle sticks and SMA(Simple Moving Average) must be used for strategy and Risk Management must be followed.
In this example I am talking about SMA 44 but it can be even SMA 24 or whatever optimized value we get for a particular stock.

**Comment by kieran-mackle** [comment]
@rahulmr I have just added this strategy for you. It is ready to run using the script below (I have also included the command to optimise the SMA period).

As per the current parameters in the config file (`sma_momentum.yaml`), each trade will risk 1% of the account, based on a stop loss set below the low of the signal candle, with some buffer set by the `exit_buffer` parameter. The risk-to-reward ratio is set by the `RR` parameter, currently 4. The candle lookback parameter is used to control how many candles backwards are considered to ensure there is upward momentum.

```py
from autotrader.autotrader import AutoTrader

at = AutoTrader()
at.configure(show_plot=True, verbosity=1)
at.add_strategy('sma_momentum')
at.backtest(start = '10/12/2021',
            end = '19/12/2021',
            initial_balance=1000,
            leverage = 30)
at.optimise(opt_params=['sma_period'],
            bounds=[(20, 60)])
at.run()
```
